data/brands/shop/bakery.json

```diff
@@ -417,7 +417,7 @@
     {
       "displayName": "BreadTalk",
       "id": "breadtalk-96a6b3",
-      "locationSet": {"include": ["sg"]},
+      "locationSet": {"include": ["142", "150"]},
```

Comment:
Wikipedia usually lists all countries and regions where stores exist, but this does not mean that all of them must be added completely, because there may be only one or two scattered stores in some countries and regions, and the changes over time are great. It is not wise to include all of Europe and Asia, my suggestion is to add only a few regions where chain store networks are clearly present, such as Hong Kong, Mainland of China, Thailand, Malaysia and etc.

[![图片](https://user-images.githubusercontent.com/42690037/224942185-81d09f40-875e-48c4-baaa-3502ed3ea8b9.png)](https://location-conflation.com/?locationSet=%7B+include%3A+%5B%27142%27%2C+%27150%27%5D+%7D%0A)